decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/moderations_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
```

Comment:
Yes, I'll review everything so that categories & moderations stay fully in core.